lib/src/executable/compile_stylesheet.dart

```diff
@@ -174,6 +174,8 @@ String _writeSourceMap(
     url = p.toUri(p.relative(sourceMapPath, from: p.dirname(destination)));
   }
 
+  var escapedUrl = url.toString().replaceAll("*/", "*\\/");
```

Comment:
This should be URL encoded, not backslash escaped (same below)

Reply:
the issue is not about escaping for usage in a URL. It is about escaping it for usage in a CSS comment. This applies *CSS* escaping

Reply:
After looking into it, I realize that URL encoding won't work with data URLs (since the percent encoding then ends up in the resulting contents), but I'm not sure backslash escaping will work either.

Reply:
Actually, nevermind. Percent encoding does work, I was just doing it wrong. `data:,/*comment%2A/` correctly resolves to `/*comment*/`, so the answer here is probably `var escapedUrl = url.toString().replaceAll('*/', '%2A/');`